amarillo/plugins/gtfs_export/router.py

```diff
@@ -53,12 +68,25 @@ async def get_file(region_id: str, user: str = Depends(verify_admin_api_key)):
                 status.HTTP_400_BAD_REQUEST: {"description": "Bad request, e.g. because format is not supported, i.e. neither protobuf nor json."}
         }
     )
-async def get_file(region_id: str, format: str = 'protobuf', user: str = Depends(verify_admin_api_key)):
+async def get_file(region_id: str, format: str = 'protobuf', requesting_user: User = Depends(get_current_user)):
+    verify_permission("gtfs", requesting_user)
     _assert_region_exists(region_id)
     if format == 'json':
-        return FileResponse(f'data/gtfs/amarillo.{region_id}.gtfsrt.json')
+        file_path = f'data/gtfs/amarillo.{region_id}.gtfsrt.json'
     elif format == 'protobuf':
-        return FileResponse(f'data/gtfs/amarillo.{region_id}.gtfsrt.pbf')
+        file_path = f'data/gtfs/amarillo.{region_id}.gtfsrt.pbf'
     else:
         message = "Specified format is not supported, i.e. neither protobuf nor json."
-        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=message)
\ No newline at end of file
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=message)
+    
+    if is_cached(file_path):
```

Comment:
GTFS-RT stand for GFTFS-Realtime. This feed should have a max time to live of 60 secs. I.e. it must not be cached for a whole days

Reply:
Fixed, I have removed caching for GTFS-RT

Reply:
This is a behavior change in contrast to before. Before, it was re-generated once per minute (which would correspond to coaching of one minute). As this file can be requested very often, it should be cached for a minute.

Reply:
Okay, I've added caching of GTFS-RT files for 1 minute.